code/modules/power/supermatter/supermatter_process.dm

```diff
@@ -214,6 +219,15 @@
 			removed.gases[/datum/gas/oxygen][MOLES] -= consumed_carbon_dioxide * 0.5
 			removed.gases[/datum/gas/pluoxium][MOLES] += consumed_carbon_dioxide * 0.25
 
+	//Very high energy radiaton will convert helium into tritium.
+	if(gas_comp[/datum/gas/helium] && power > POWER_PENALTY_THRESHOLD)
+		var/helium_pp = env.return_pressure() * gas_comp[/datum/gas/helium]
+		var/consumed_helium = clamp(((helium_pp - HELIUM_CONSUMPTION_PP) / (helium_pp + HELIUM_PRESSURE_SCALING)) * gasmix_power_ratio * clamp((power - POWER_PENALTY_THRESHOLD) / POWER_PENALTY_THRESHOLD, 0, 1), HELIUM_CONSUMPTION_RATIO_MIN, HELIUM_CONSUMPTION_RATIO_MAX)
+		consumed_helium = min(consumed_helium * gas_comp[/datum/gas/helium] * combined_gas, removed.gases[/datum/gas/helium][MOLES])
+		if(consumed_helium)
+			removed.gases[/datum/gas/helium][MOLES] -= consumed_helium
+			removed.gases[/datum/gas/tritium][MOLES] += consumed_helium * 0.5
```

Comment:
Do you think this is something people will actually use?

Reply:
Probably in longer rounds when managing gases becomes more important, but in shorter rounds it's not that useful.

Reply:
why should we add it then?

Reply:
It allows a way to quickly but dangerously recycle helium back into tritium after using tritium in HFR. Might be interesting for people that want to try and conserve their gases and not waste them.